lib/plugins/aws/info/getStackInfo.js

```diff
@@ -28,7 +28,11 @@ module.exports = {
           if (result) stackData.outputs = result.Stacks[0].Outputs;
         }),
     ];
-    if (this.serverless.service.provider.httpApi && this.serverless.service.provider.httpApi.id) {
+    if (
+      this.serverless.service.provider.httpApi &&
+      this.serverless.service.provider.httpApi.id &&
+      typeof this.serverless.service.provider.httpApi.id === 'string'
+    ) {
```

Comment:
Let's remove this change. Currently we accept just directly passed HTTP API ids. If that's need to be enhanced with support for some other formats, it should be in scope of other issue/pr